code/modules/flufftext/Hallucination.dm

```diff
@@ -1294,3 +1295,29 @@ GLOBAL_LIST_INIT(hallucination_list, list(
 	H.preparePixelProjectile(target, start)
 	H.fire()
 	qdel(src)
+
+/datum/hallucination/sleeping_carp
+
+/datum/hallucination/sleeping_carp/New(mob/living/carbon/C, forced = TRUE)
+	set waitfor = FALSE
+	..()
+	var/list/mobsyup = list()
+	for (var/mob/living/carbon/A in orange(C,2))
+		if (get_dist(C,A) < 2 && C != A)
```

Comment:
```suggestion
		if (get_dist(C,A) < 2)
```
should be impossible since orange doesn't check the epicenter turf.

Reply:
It does